keyboards/handwired/polykybd/split72/keymaps/default/lang/named_glyphs.h

```diff
@@ -272,6 +272,7 @@
 #define TECHNICAL_ALTERNATIVE       	u"\x2387"
 #define TECHNICAL_CONTROL           	u"\x2388"
 #define TECHNICAL_ESCAPE            	u"\x238B"
+#define PRINT_SCREEN                	u"\x2399"
```

Comment:
This change should ideally come from the lang_lut.xlsx, otherwise it get overwritten the next time we run cog

Reply:
Btw... just good to know: There are 2 other print screen glyphs, which are PRIVATE_IMAGE (like a picture as I wanted to get away of the concept of actually printing something) and PRIVATE_PRINTER (also looks like a printer, maybe less stylish). But it is all good having just another one.

Reply:
this is from updating the lang_lut.xlsx and running cog

Reply:
Alright, looks like I just didn't see langlut.xlsx